rs/tests/testnets/small_high_perf.rs

```diff
@@ -1,6 +1,7 @@
-// Set up a testnet containing:
+// Set up a small high performance testnet containing:
 //   one 1-node System and one 1-node Application subnets, single boundary node, and a p8s (with grafana) VM.
-// All replica nodes use the following resources: 64 vCPUs, 480 GiB of RAM, and 2'000 GiB disk.
+// All replica nodes use the following resources: 6 vCPUs, 24 GiB of RAM, and 50 GiB disk.
+// Application subnet node is: 64 vCPUs, 512 GiB of RAM, and 500 GiB disk.
```

Comment:
I think the following is a bit clearer:
```suggestion
// The single system subnet node uses the default: 6 vCPUs, 24 GiB of RAM, and 50 GiB disk while the 
// single application subnet node uses: 64 vCPUs, 512 GiB of RAM, and 500 GiB disk.
```